src/cloudinary-image.component.spec.ts

```diff
@@ -524,4 +526,155 @@ describe('CloudinaryImage', () => {
       expect(img.attributes.getNamedItem('src').value).toEqual(jasmine.stringMatching(/c_scale,dpr_auto,w_auto/));
     });
   });
+
+  describe('lazy load image', async () => {
+    @Component({
+      template: `
+          <div class="startWindow"><cl-image loading="lazy" width="300" public-id="bear"></cl-image></div>
+          <div style="margin-top: 300px"><cl-image loading="lazy" width="300" public-id="bear"></cl-image></div>
+          <div style="margin-top: 300px"><cl-image loading="lazy" width="300" public-id="bear"></cl-image></div>
+          <div style="margin-top: 300px"><cl-image loading="lazy" width="300" public-id="bear"></cl-image></div>
+          <div style="margin-top: 300px"><cl-image loading="lazy" width="300" public-id="bear"></cl-image></div>
+          <div style="margin-top: 300px"><cl-image loading="lazy" width="300" public-id="bear"></cl-image></div>
+          <div style="margin-top: 300px"><cl-image loading="lazy" width="300" public-id="bear"></cl-image></div>
+          <div class="endWindow" style="margin-top: 300px"><cl-image loading="lazy" width="300" public-id="bear"></cl-image></div>`
+    })
+    class TestComponent {}
```

Comment:
@strausr , why is that part of this PR?

Reply:
All the changes for the Lazy load are here as well so that they can be tested with the placeholder. Should I remove it?

Reply:
@strausr , lazy load should be a different PR. 
Wasn't is already merged?